R/lsmeans.R

```diff
@@ -2,30 +2,34 @@
 #' Least Square Means
 #'
 #'
-#' Estimates the least square means from a linear model. This is done by
-#' generating a prediction from the model using an hypothetical observation
-#' that is constructed by averaging the data. See details for more information.
+#' Estimates the least square means from a linear model. The exact implementation
+#' / interpretation depends on the weighting scheme; see details for more information.
 #'
 #' @details
 #'
-#' The lsmeans are obtained by calculating hypothetical patients
-#' and predicting their expected values. These hypothetical patients
-#' are constructed by expanding out all possible combinations of each
-#' categorical covariate and by setting any numerical covariates equal
-#' to the mean.
+#' For `.weights = "proportional"` (the default) the lsmeans are obtained by
+#' taking the average of the fitted values for each patient.
+#' In terms of the interactions between two continuous covariates 
+#' this is equivalent to constructing a hypothetical patient whose
+#' interaction term is `mean(X * Y)`. This is opposed to the
+#' `emmeans` package which calculates the interaction term
+#' of the hypothetical patient as `mean(X) * mean(Y)`. The approach
+#' outlined here is equivalent to standardization or g-computation.
```

Comment:
I would put "The approach outlined here is equivalent to standardization or g-computation." above the sentence "In terms of the interactions..".